lib/teamlab/modules/crm/crm_opportunities.rb

```diff
@@ -23,6 +23,13 @@ def get_all_opportunity_contacts(opportunity_id)
       @request.get(['opportunity', opportunity_id.to_s, 'contact'])
     end
 
+    # Returns a list of all the opportunities for the contact with the ID specified in the request
+    # @param contact_id [String] Contact ID
```

Comment:
I think Contact id can be - string and integer, since on 30 line `contact_id.to_s` used

So
```
    # @param contact_id [String, Integer] Contact ID
```